suite-common/transaction-cache-engine/src/TransactionCacheEngine.ts

```diff
@@ -119,6 +119,7 @@ export class TransactionCacheEngine {
             const result = await TrezorConnect.getAccountInfo({
                 coin,
                 descriptor,
+                // TODO add identity when used for eth-like coins
```

Comment:
ok, so we are creating this fix for that ethereum-head with many passphrases and he will still get banned when using mobile app :D 

what is the reason for not having it here?

Reply:
a) I'm lazy, Ethereum is trash and it's too much work (identities must be stored inside `TransactionCacheEngine` together with descriptors as it doesn't know anything about devices and device states), but more importantly

b) `TransactionCacheEngine` currently supports only XRP (see e.g. https://github.com/trezor/trezor-suite/blob/develop/suite-native/accounts/src/useTransactionCache.ts#L12-L13) so it seemed unnecessary to add the identities yet.